raiden-dapp/src/components/overlays/InfoOverlay.vue

```diff
@@ -38,60 +40,77 @@ export default class InfoOverlay extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/dimensions';
 @import '@/scss/mixins';
 @import '@/scss/colors';
 
 .info-overlay {
-  align-items: center;
   background-color: $transfer-screen-bg-color;
   border-radius: 10px;
-  display: flex;
-  flex-direction: column;
-  height: calc(844px);
-  padding: 0 46px;
+  height: 844px;
   position: absolute;
   width: 100%;
   z-index: 102;
+
   @include respond-to(handhelds) {
     border-radius: 0;
     height: 100vh;
+    overflow-y: auto;
```

Comment:
Can't this be device inspecific? So it just applies when necessary. And as it should be only become necessary for very small devices this is implicit.

Reply:
Yes, I'll fix that 👍🏽